api/AudioBuffer.json

```diff
@@ -0,0 +1,659 @@
+{
+  "api": {
+    "AudioBuffer": {
+      "AudioBuffer": {
+        "__compat": {
```

Comment:
Please add a description field indicating that it is the constructor (not obvious from the id)